app/src/main/res/layout/recently_played_fragment.xml

```diff
@@ -47,7 +47,8 @@
         android:overScrollMode="never"
         android:paddingTop="8dp"
         android:paddingBottom="172dp"
-        android:scrollbars="none" />
+        android:scrollbars="none"
+        android:tag="recently_played_recycler_view" />
```

Comment:
Why do we need this tag?

Reply:
It was added in a middle change that was canceled later.  
That is adding robolectric tests for the span count for each configuration, but I have modified the test cases themselves instead.  
I've removed the tag. PTAL

Reply:
OKay. Thanks.